app/components/Base/TipsBase.js

```diff
@@ -37,3 +37,13 @@ export default BaseRoundedNoPadding.extend`
     }
   }
 `;
+
+export const TipsBaseSmallPadding = TipsBase.extend`
```

Comment:
use emotion css and separate the components into different files

Reply:
Done